llvm-to-smt/llvm-passes/LLVMToSMT/FunctionEncoder.cpp

```diff
@@ -2044,9 +2050,13 @@ void FunctionEncoder::populateBBAssertionsMap(BasicBlock &B) {
       handleStoreInst(*dyn_cast<StoreInst>(&I));
     } else if (isa<CallInst>(&I)) {
       handleCallInst(*dyn_cast<CallInst>(&I));
+    } else if (isa<BranchInst>(&I)) {
+      handleBranchInst(*dyn_cast<BranchInst>(&I), BBAssertionsMapPass);
+    } else if (isa<ReturnInst>(&I)) {
+      handleReturnInst(*(dyn_cast<ReturnInst>(&I)), BBAssertionsMapPass);
```

Comment:
If we know nothing should be done for these, we could also explicitly do nothing here. I don't know what is better/clearer.

Reply:
I am okay either way, but I just added a call because it is clear we are "handling" the instructions by calling a `handle*` function. This function `populateBBAssertionsMap()` should iterate over all supported instructions.